src/components/Footer/Footer.tsx

```diff
@@ -1,21 +1,26 @@
 import React from 'react';
 import { Typography, Container } from '@mui/material';
-import BoxNew from './Footer.style';
+import { useTranslation } from 'react-i18next';
+import Wrapper from './Footer.style';
 
-const Copyright = () => (
-  <Typography variant="body2" color="text.secondary">
-    {'Copyright © '}
-    {new Date().getFullYear()}
-  </Typography>
-);
+const Copyright = () => {
+  const [t] = useTranslation();
+
+  return (
+    <Typography variant="body2" color="text.secondary">
+      {t('footer:copyright')}
+      2023
```

Comment:
prop for `footer:copyright`

Reply:
Can you explain me, what prop for footer:copyright means. May I do like that ? 
    <Typography variant="body2" color="text.secondary">
      {t('footer:copyright', { text: '\u00a9' })}
      2023
    </Typography>

  footer: {
    copyright: 'Copyright {{text}} ',
  },

Reply:
copyright: `Copyright \u00a9 {{ year }}}`

Reply:
May I deal with efficiency like that ? 

const Copyright = () => {
  const [t] = useTranslation();
  const [date, setDate] = useState<number | null>(null);

  const getYear = () => setDate(new Date().getFullYear());

  useEffect(() => {
    getYear();
  }, []);

  return (
    <Typography variant="body2" color="text.secondary">
      {t('footer:copyright', { year: date })}
    </Typography>
  );
};

Reply:
no.

You should make a constant with useMemo for year property

Reply:
add to check improve create-layout-compo. and add Lang.Selector